src/main/resources/export/vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This is invalid JSON.
JSON can't include comments.

Reply:
@JLLeitschuh This is a vscode file, which allows comments

Reply:
Lordy. Fine.

Reply:
I've looked over the latest version of the fork.  It all looks good to me.